app/src/main/java/org/thoughtcrime/securesms/ui/theme/Colors.kt

```diff
@@ -49,4 +49,6 @@ val dangerLight = Color(0xFFE12D19)
 val disabledDark = Color(0xFFA1A2A1)
 val disabledLight = Color(0xFF6D6D6D)
 
+val warningUniversal = Color(0xFFFCB159)
```

Comment:
This is already declared as PrimaryOrange. Not sure if it needs another declaration as this is colors definitions, which can then be used as different things by themes, ie: warning, bg, etc...

Reply:
This one is no longer used. Deleted